### ISSUE TITLE
Scaling System doesn't work

### ISSUE THREAD

**Issue by ImmortalusDev** [TrinityCore/TrinityCore#24771]
Every spell does very low damage.

I tested it even with Classic spells.

**Expected behaviour:**

Spells should be scaled depends on caster lvl and target lvl.

**Steps to reproduce the problem:**

1. .cast 304829 on 120 level creature/player.
2. or make simple smartscript which allow creature to cast 304829 on you
3. notice very low damage values

**Branch(es):**
master

**TC rev. hash/commit:** 
TrinityCore rev. 8a5321c2fc8d

**Operating system:** 
Win 10 x64

**Comment by tails2k** [comment]
Also here. Master Branch, Checkt Out yesterday.
Steps to reproduce the problem:
1: Creat a warlock
2: go kill any
3: you spells dont count up

I have a level 6 warlock and the main spell make 5 Damage also i am at level 1. It dosent count up.

Operating system:
Debian 10

**Comment by Faq** [comment]
@tails2k edit your post by providing commit hash

**Comment by MaxtorCoder** [comment]
Hasn't this been reported before, or was that for Creature level.

**Comment by tails2k** [comment]
> 
> 
> @tails2k edit your post by providing commit hash

@Faq Edit it. I make every night an auto Update of the Server.

**Comment by tails2k** [comment]
> 
> 
> Hasn't this been reported before, or was that for Creature level.

@MaxtorCoder yes. It say" github-actions bot added Branch-master Invalid-IncompleteData/OrNotTrinityCore labels on 8 Jun 2020" from "ImmortalusDev commented on 8 Jun 2020 •"
Sooo when i can help say me what to do. When its IncompleteData... i would say that he is not the only one how has this error.

**Comment by MaxtorCoder** [comment]
> > Hasn't this been reported before, or was that for Creature level.
> 
> @MaxtorCoder yes. It say" github-actions bot added Branch-master Invalid-IncompleteData/OrNotTrinityCore labels on 8 Jun 2020" from "ImmortalusDev commented on 8 Jun 2020 •"
> Sooo when i can help say me what to do. When its IncompleteData... i would say that he is not the only one how has this error.

IncompleteData simply means that there isn't enough data

**Comment by funjoker** [comment]
8a5321c2fc8d <--- incomplete hash

**Comment by tails2k** [comment]
> 
> 
> > > Hasn't this been reported before, or was that for Creature level.
> > 
> > 
> > @MaxtorCoder yes. It say" github-actions bot added Branch-master Invalid-IncompleteData/OrNotTrinityCore labels on 8 Jun 2020" from "ImmortalusDev commented on 8 Jun 2020 •"
> > Sooo when i can help say me what to do. When its IncompleteData... i would say that he is not the only one how has this error.
> 
> IncompleteData simply means that there isn't enough data

@MaxtorCoder can i give you more anyway? i would help when i can

**Comment by Aokromes** [comment]
this issue is still valid?

**Comment by ImmortalusDev** [comment]
It is fine on clean TC, looks like it.
